openedx/core/lib/xblock_utils.py

```diff
@@ -306,6 +306,7 @@ def add_staff_markup(user, has_instructor_access, disable_staff_debug_info, cour
 
         not_ccx_course = CustomCourseForEdX.objects.filter(course_id=course_id)
         # will not return the View Unit in Studio button when viewing a CCX course
+        # this means the user cannot edit the course on Studio
```

Comment:
@felipevolpone, I'd still like to see more work on this before approving it. For example, I'd like an affirmative variable name for the variable (`is_ccx_course`) or similar, followed by something like `if is_studio_course and not is_ccx_course` for the conditional. Also, on second look, I think you might have a logic error that's masked by the variable name you're using. I would additionally take a look at the Django `QuerySet.exists()` method as a way to explicitly get a boolean from a query.

Finally, I'd like a comment describing _why_ the behavior is this way, rather than a comment simply describing what the behavior _is_. You can refer to OSPR-1383 for help in drafting this comment.